app/models/ability.rb

```diff
@@ -75,6 +75,8 @@ def signed_in(user)
     can [:new, :create], Payment, user_id: user.id
     can [:index, :show], PhysicalTicket, user: user
 
+    can [:new, :create, :update, :index, :show], Booth
```

Comment:
A user should be able to access the actions of the booths for which the user is a submitter or a responsible. We should limit that can definition accordingly.